plugin/src/main/kotlin/com/github/triplet/gradle/play/PlayPublisherPlugin.kt

```diff
@@ -117,56 +117,49 @@ class PlayPublisherPlugin : Plugin<Project> {
                     "Uploads all Play Store metadata for $variantName."
             ) {
                 init()
-                resDir = playResourcesTask.resDir
+                resDir = playResourcesTask.get().resDir
 
                 dependsOn(playResourcesTask)
-                publishListingAllTask.dependsOn(this)
-
-                // Remove in v3.0
-                val new = this
-                project.newTask<Task>("publishListing$variantName", "", null) {
-                    dependsOn(new)
-                    doFirst { logger.warn("$name is deprecated, use ${new.name} instead") }
-                }
+            }
+            publishListingAllTask.configure { dependsOn(publishListingTask) }
+            // TODO Remove in v3.0
+            project.newTask("publishListing$variantName", "", null) {
+                dependsOn(publishListingTask)
+                doFirst { logger.warn("$name is deprecated, use ${publishListingTask.get().name} instead") }
             }
 
             val processPackageMetadata = project.newTask<ProcessPackageMetadata>(
                     "process${variantName}Metadata",
                     "Processes packaging metadata for $variantName.",
                     null
-            ) {
-                init()
-
-                variant.checkManifest.dependsOn(this)
-            }
+            ) { init() }
+            checkManifest.dependsOn(processPackageMetadata)
```

Comment:
Should this be `checkManifest.configure { dependsOn(processPackageMetadata) }`?

Reply:
`checkManifest` actually comes from the AGP and they haven't migrated to the new API yet. But yeah, once they do, that's what it will look like. 👍